hs_core/views/utils.py

```diff
@@ -536,7 +537,8 @@ def create_form(formclass, request):
     return params
 
 
-def get_my_resources_list(request):
+# TODO : delete this function
+def get_my_resources_list_old(request):
```

Comment:
Are you leaving this for our sake as a part of the review?

Reply:
I left it for my testing to see the difference between the old query and the new one. In my final PR I will delete it.